okta/provider_sweeper_test.go

```diff
@@ -0,0 +1,75 @@
+package okta
```

Comment:
Whats this sweeper thing? My understanding is that you can write your tests in a way that this doesn't happen?

Your tests should destroy after, and actually test to ensure it happened? https://github.com/articulate/terraform-provider-kong/blob/master/kong/resource_kong_api_test.go#L24

Reply:
![image](https://user-images.githubusercontent.com/145406/45779803-e575c780-bc10-11e8-92e4-fc76b620dab4.png)

Reply:
https://www.terraform.io/docs/extend/testing/acceptance-tests/sweepers.html

The sweepers are supposed to be a failsafe for leaks. They were easy to implement and I saw a lot of providers implementing them so I added it as well. And yea totally, the old destroy logic also needs work, haven't gotten to refactoring this yet. It works fine with the happy path but I was seeing leaks in certain cases. I will fix that!

Reply:
Ahh I see, so this is to handle the case where a previous test may have failed and not cleaned up.

Reply:
Yea sweepers are a failsafe to avoid dangling resources, which I was seeing a lot of when tests would fail. I will take a bit of time on this PR and try fixing the destroy logic so it works on failure as well, if that is possible. I bet it is.